lib/fastlane/plugin/revenuecat_internal/helper/revenuecat_internal_helper.rb

```diff
@@ -9,9 +9,18 @@
 
 module Fastlane
   UI = FastlaneCore::UI unless Fastlane.const_defined?(:UI)
+  SUPPORTED_PR_LABELS = %w[breaking build ci docs feat fix perf refactor style test].to_set
 
   module Helper
     class RevenuecatInternalHelper
+      @cached_commits_by_old_version = {}
+      @pr_resp_items_by_sha = {}
```

Comment:
Hmm so these are instance variables. Considering all methods in this helper are class methods (basically `static`), I think these should be class variables as well. To do that you should add 2 `@`, Like:
```suggestion
      @@cached_commits_by_old_version = {}
      @@pr_resp_items_by_sha = {}
```
It does work with this according to this answer: https://stackoverflow.com/a/9311665. But I feel these would be better like that.

A different argument would be to change these to not be class methods and create instances of the helper inside the actions... I followed this convention since it's how it's done in fastlane afaik, so I think it's better to leave as is.

Reply:
I did them class instance vars because rubocop and RubyMine complains if I make them class vars:

```
 Style/ClassVars: Replace class var @@pr_resp_items_by_sha with a class instance var.
 ```
 
 So I figured that's what I should be doing. Do you think I should still do class vars?

Reply:
Hmm interesting... https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/ClassVars
I trust the default rubocop rules, and considering it works like this i think we should leave it as you have it, but I don't know if this explanation warranted disabling a whole language feature... 🤷‍♂️